decentraland/unity-renderer
# fix: avatar shape glitch caused by zero scale

## What does this PR change?

Closes #2997 

## How to test the changes?

- Download and extract this scene: [mirrors.zip](https://github.com/decentraland/unity-renderer/files/9550998/mirrors.zip)
- Enter the folder on a terminal
- Initialize the scene using `dcl start` ([Installation Guide](https://docs.decentraland.org/development-guide/SDK-101/) of the DCL Tools)
- When the browser opens add this `&renderer-branch=fix/avatar-shape-glitch` to the end of the url
- Url should look like this http://127.0.0.1:8000/?position=0%2C0&SCENE_DEBUG_PANEL&renderer-branch=fix/avatar-shape-glitch
- Check that there is no glitch

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
